bedrock/security/tests/test_views.py

```diff
@@ -125,9 +125,9 @@ def test_latest_queryset_product(self):
 
     def test_latest_queryset_product_version(self):
         """Should advisories for a single product version."""
-        advisories_30 = [self.new_advisory(fixed_in=['Firefox 30.{0}'.format(i)])
+        advisories_30 = [self.new_advisory(fixed_in='Firefox 30.{0}'.format(i))
                          for i in range(5)]
-        advisories_29 = [self.new_advisory(fixed_in=['Firefox 29.0.{0}'.format(i)])
+        advisories_29 = [self.new_advisory(fixed_in='Firefox 29.0.{0}'.format(i))
```

Comment:
Yes. I was lazy and didn't add a specific test, but I changed these lines and left others in this test class as a list. Those are the two primary types used and I wanted to get this in before another advisory landed.